pinot-controller/src/main/java/org/apache/pinot/controller/helix/core/PinotHelixResourceManager.java

```diff
@@ -168,8 +168,8 @@ public PinotHelixResourceManager(@Nonnull ControllerConf controllerConf) {
   /**
    * Create Helix cluster if needed, and then start a Pinot controller instance.
    */
-  public synchronized void start() {
-    _helixZkManager = registerAndConnectAsHelixParticipant();
+  public synchronized void start(HelixManager helixZkManager) {
+    _helixZkManager = helixZkManager;
```

Comment:
Use helixManager.getInstanceName() as the _instanceId

Reply:
Done